cdap-ui/app/hydrator/leftpanel.less

```diff
@@ -309,9 +309,9 @@ body.theme-cdap.state-hydrator-create {
         }
         div.side-panel.top {
           .hydrator-metadata {
-            width: ~"calc(100vw - 755px)";
-            width: ~"-moz-calc(100vw - 755px)";
-            width: ~"-webkit-calc(100vw - 755px)";
+            width: ~"calc(100vw - 805px)";
```

Comment:
Seems like the problem was there even when the value was 755px. I had to modify a different value to fix this.